src/content/structured/components/tree-view/accessibility.mdx

```diff
@@ -0,0 +1,70 @@
+---
+path: "/components/tree-view/accessibility"
+
+navPriority: 41
+
+date: "2024-08-23"
+
+title: "Tree view"
+
+status: "CANARY"
+
+subTitle: "Tree view provides a navigation tree for use in applications with hierarchical organised pages."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/tree-view" },
+    { title: "Code", path: "/components/tree-view/code" },
+    { title: "Accessibility", path: "/components/tree-view/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+The tree view items will focus top-down, including the child nodes of any expanded parent items.
+Tree view items can be iterated over with the tab or up and down keys.
+Enter and space keys can be used to expand the currently selected parent node, or to follow the link of the currently selected node.
+
+## For Assistive Technology
+
+Tree view items consist of a button and an extended text area. When using a screen reader, the title of the tree view will be declared first along with how many items it contains.
+When focussed on a tree view item, screen readers will state its:
+
+<ul>
+  <li>Label</li>
+  <li>Index</li>
+  <li>State</li>
+  <li>Number of nested items</li>
+</ul>
+Any tree view items which aren’t in focus will not be declared by the screen reader.
+
+## Based on
+
+The tree view component has been based on the following resources:
+
+<ul>
+  <li>
+    <ic-link
+      target="_blank"
+      href="https://carbondesignsystem.com/components/tree-view/usage/"
+      rel="noreferer noopener nofollow"
+    >
+      Carbon Design System - Tree View
+    </ic-link>
+    , Carbon Design System IBM Group, accessed February 2024.
+  </li>
+  <li>
+    <ic-link
+      target="_blank"
+      href="https://v4.clarity.design/tree-view#top"
+      rel="noreferer noopener nofollow"
+    >
+      Project Clarity - Tree View
+    </ic-link>
+    , VMWare Clairty, accessed September 2023.
+  </li>
+</ul>
```

Comment:
This section is should be used for links to resources on accessibility and research that supports the design approach. We don't typically refer to other design systems in this section especially in a component like this where there's somewhat of a standardised design for tree view. Are there any accessibility resources you used for research?